client/src/App.jsx

```diff
@@ -26,10 +29,12 @@ class Photos extends React.Component {
       productId: null,
       primaryPhotoUrl: null,
       productPhotosUrls: [],
-      modalCordinates: {x: 0, y: 0}
```

Comment:
Spell check "cordinates" should be "coordinates".

Reply:
🤠